modules/telco-upgrade-starting-the-cluster-upgrade.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/day_2_core_cnf_clusters/upgrading/telco-upgrade-completing-the-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="telco-upgrade-starting-the-cluster-upgrade_{context}"]
+= Starting the cluster upgrade
+
+When upgrading from one y-stream release to the next, you need to make sure that the intermediate z-stream releases are also compatible.
+
+[NOTE]
+====
+You can verify that you are upgrading to a viable release by running the `oc adm upgrade` command.
+This command lists the compatible upgrade releases.
+====
+
+.Procedure
+. Start the upgrade:
++
+[source,terminal]
+----
+$ oc adm upgrade --to=4.15.33
+----
++
+.Example output
+[source,terminal]
+----
+Requested update to 4.15.33
+----
++
+This will change depending on your particular upgrade:
```

Comment:
When you say "this", are you talking about this output, or talking about what you enter in the command?

If it's the command, maybe you should mention that in the command, either with a callout or making the value replaceable.